EkaterinaUlitina/src/main/java/HomeWorkFromLesson6/CopyIntArray.java

```diff
@@ -0,0 +1,27 @@
+package HomeWorkFromLesson6;
+
+public class CopyIntArray {
+    public static void main(String[] args) {
+
+        int[][] arr = new int[100][100];
```

Comment:
Suggest to extract size(100) to separate variable.

Reply:
Done